atd-vze/src/views/Locations/LocationMap.js

```diff
@@ -60,10 +60,7 @@ const LocationMap = ({ data }) => {
         <Layer {...polygonDataLayer} />
       </Source>
       {/* add nearmap raster source and style */}
-      <LabeledAerialSourceAndLayer
-        beforeId="location-polygon"
-        showMockLayerLocally
-      />
+      <LabeledAerialSourceAndLayer beforeId="location-polygon" />
```

Comment:
Ended up disabling the blue test layer since we have a workaround until we get our VZ key updated by CTM. Maybe I should just remove the test layer altogether in this PR? 🤔

Reply:
i guess i'm on board with removing it! your call 🙏